index.html

```diff
@@ -412,6 +412,27 @@ <h3>Transcode to H.265/HEVC</h3>
     </div>
     <!-- ends Transcode to H.265 -->
 
+    <!-- Transcode to Ogg/Theora -->
+    <label class="recipe" for="transcode_ogg">Transcode to an Ogg Theora</label>
+    <input type="checkbox" id="transcode_ogg">
+    <div class="hiding">
+      <h3>Transcode to H.265/HEVC</h3>
```

Comment:
Looks like this should say `Transcode to Ogg/Theora`

Reply:
Indeed! PR https://github.com/amiaopensource/ffmprovisr/pull/334 should fix